.github/workflows/ci-pytorch-dockers.yml

```
@@ -8,12 +8,9 @@ on:
     types: [opened, reopened, ready_for_review, synchronize]  # added `ready_for_review` since draft is skipped
     paths:
       - "dockers/**"
-      - "!dockers/README.md"
-      - "requirements.txt"
-      - "requirements/**"
-      - "!requirements/*/docs.txt"
+      - ".github/workflows/ci-pytorch-dockers.yml"
+      - "requirements/pytorch/*"
```

Should this be triggered by changes in lite's requirements, too?
```suggestion
      - "requirements/pytorch/*"
      - "requirements/lite/*"
```

---

Good point